src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1275,6 +1275,12 @@
   <data name="DoNotUseStackallocInLoopsMessage" xml:space="preserve">
     <value>Potential stack overflow. Move the stackalloc out of the loop.</value>
   </data>
+  <data name="InstantiateArgumentExceptionsCorrectlyChangeToTwoArgument" xml:space="preserve">
```

Comment:
Can you suffix with `Message`, `Title` or `Description` for clarity how we use this resource string?

Reply:
Ah, so the new strings are just used in the code fix and not any diagnostic message or description? Consider adding a suffix `CodeFixTitle` to the name of the resource string, such as `InstantiateArgumentExceptionsCorrectlyFlipArgumentOrderCodeFixTitle`.

Reply:
Yes it is for codefix, will add that suffix